frontend/AnimeCore/src/routes/+layout.svelte

```diff
@@ -74,31 +74,31 @@
                     component: Explore,
                     class: "w-[1.25vw] text-white"
                 },
-                url: undefined,
+                url: "/explore",
```

Comment:
change these back to undefined for now?

Reply:
Uhhh not needed? Change in  dev repo?

Reply:
Okay? or we can just add some text like?
"Page in construction/development"
Ik we cant use that effect without other pages, so what
>"Page in construction/development"

I think this would be better ( can be changed ), if a user visit, uhh what you think

Reply:
uhh do it in your project, i will just copy that over here.